src/backend/routers/case_manager.ts

```diff
@@ -17,6 +17,44 @@ export const case_manager = router({
     return result;
   }),
 
+  getMyStudentsAndIepInfo: authenticatedProcedure.query(async (req) => {
+    const { userId } = req.ctx.auth;
+
+    const studentData = await req.ctx.db
+      .selectFrom("student")
+      .where("assigned_case_manager_id", "=", userId)
+      .fullJoin("iep", (join) =>
+        join.onRef("iep.student_id", "=", "student.student_id")
+      )
```

Comment:
In light of this week's discussions, at this point it sounds to us like the best course of action at this point is to simplify this to just make it a join of iep and student on student_id, returning all where the student's assigned case manager is the user. This means not worrying about start or end dates or grabbing just one iep per student since we're operating on the assumption that there's only one iep per student. We're expecting to have a commit today that will have the revised procedure.